### DISCUSSION TITLE
Infeasible error while trying to Solve host microbe model with pFBA

### DISCUSSION THREAD

**Discussion by Ilhem36** [opencobra/cobrapy#1258] [General]
#### Problem description

Please explain:
I'm trying to find solution on a host microbe model using pFBA  but it is running since four days an then I received the error that the model is infeasible. Do you have please any idea on how to solve the problem or maybe using another type of  flux balance analysis because the model is huge ( contain human and microbial community model).
Thank you for your help.

#### Code Sample

Create a [minimal, complete, verifiable example](https://stackoverflow.com/help/mcve).

```
solution=FBA_solver("glpk", host_microbe_model)
```

```
![image](https://user-images.githubusercontent.com/75625865/183087022-43e54ec9-9147-4850-bc13-ce9f475afa11.png)

```

**Comment by Ilhem36** [comment]
![image](https://user-images.githubusercontent.com/75625865/183087063-05432e9c-2fc5-47a8-af9b-b158bcfe2f3c.png)

**Comment by cdiener** [comment]
We would need much more information to see what is going on, but infeasible means that you have incompatible constraints in your model. Like x>3 as and x<2 simultaneously for instance.

On another note, if your model has more than 100,000 reactions GLPK will become bad at solving it. Simplex methods don't perform well with models of that size. You can try with CPLEX or Gurobi and choosing an interior point method for solving the problem ("barrier" methods in CPLEX and Gurobi).

**Reply by Ilhem36** [reply]
Thank your for your answer my model contains 204417 reactions and 166751 metabolites. In fact I have merged  two models (one of them generated by micom and ) using the merge function and I'm trying to use pfba to find solution. what other information do you need to know? 
how can I please check the infeasibility and the incompatible constraints  of a model in cobra py please?
Thank you.
